### ISSUE TITLE
Duplicate references at bottom of pages

### ISSUE THREAD

**Issue by ect123** [ubc-geomatics-textbook/geomatics-textbook#14]

**Comment by ect123** [comment]
Currently this is an issue in the `ch17bib` branch but has been observed in other chapters too. Would be good to try clearing cache and rebuilding to see if it fixes it.

**Comment by nickmmurphy** [comment]
I've been trying to figure this out. I can see the duplicate references code in the HTML output, but I can't figure out how to edit it. I also don't know how it got there in the first place!

**Comment by ect123** [comment]
Yeah, this one is kind of a mystery. I do remember other chapters having this issue before but for some reason they've been resolved on their own. 

Looking at the [html doc for this chapter](https://github.com/ubc-geomatics-textbook/geomatics-textbook/blob/develop/docs/making-beautiful-maps.html) in the docs folder, a duplicate `refs` container is made.

The [Pandoc docs suggest](https://pandoc.org/MANUAL.html#placement-of-the-bibliography) the `refs` container should only be at the bottom of the page, unless a "div with id refs" already exists, which doesn't seem to be the case. I think this might be something with my system used to knit the html doc, because I don't see anything wrong with the RMarkdown file or the citations.

This Issue seems like it might be related: [https://github.com/crsh/papaja/issues/121](https://github.com/crsh/papaja/issues/121). @nickmmurphy can you update your rmarkdown package and try knitting Chapter 17 to see if you get duplicate citations?

**Comment by nickmmurphy** [comment]
Hi @ect123, I already had rmarkdown 2.1.3, which I believe is the most current package. I updated my R version to 4.1, but the issue seems to remain.
